tools/exporter/csvExporter.py

```diff
@@ -68,23 +135,47 @@ def audit_items(self):
         """
         return self.audit_json['header_items'] + self.audit_json['items']
 
+    def audit_custom_response_id_to_label_map(self):
+        """
+        :return:     dictionary mapping custom response_id's to their label
+        """
+        custom_response_sets = self.audit_json['template_data']['response_sets']
+        audit_custom_response_id_to_label_map = dict()
+        for response_set in custom_response_sets.keys():
+            for response in custom_response_sets[response_set]['responses']:
+                audit_custom_response_id_to_label_map[response['id']] = response['label']
+        return audit_custom_response_id_to_label_map
+
     def common_audit_data(self):
         """
         :return:    Selected sub-properties of the audit_data property of the audit JSON as a list
         """
         audit_data_property = self.audit_json['audit_data']
+        template_data_property = self.audit_json['template_data']
         audit_data_as_list = list()
         audit_data_as_list.append(audit_data_property['authorship']['owner'])
         audit_data_as_list.append(audit_data_property['name'])
         audit_data_as_list.append(audit_data_property[SCORE])
         audit_data_as_list.append(audit_data_property['total_score'])
         audit_data_as_list.append(audit_data_property[SCORE_PERCENTAGE])
         audit_data_as_list.append(audit_data_property['duration'])
-        audit_data_as_list.append(audit_data_property['date_started'])
-        audit_data_as_list.append(audit_data_property['date_completed'])
+        audit_data_as_list.append(self.reformat_date(audit_data_property['date_started']))
+        audit_data_as_list.append(self.reformat_date(audit_data_property['date_completed']))
         audit_data_as_list.append(self.audit_id())
+        audit_data_as_list.append(self.audit_json['template_id'])
+        audit_data_as_list.append(template_data_property['metadata']['name'])
+        audit_data_as_list.append(template_data_property['authorship']['author'])
         return audit_data_as_list
 
+    def reformat_date(self, date):
+        """
+        :param date:    date in the format: 2017-03-03T03:45:58.090Z
```

Comment:
A comment about 'why' we reformat would help here. Or name the function convert_to_user_friendly_date(self, iso_date).